Makefile

```diff
@@ -276,7 +276,7 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST) $(KUBECTL) $(KIND) ## Create
 	./hack/create-custom-cloud-provider-config.sh
 
 	# Deploy CAPI
-	curl --retry $(CURL_RETRIES) -sSL https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.1.4/cluster-api-components.yaml | $(ENVSUBST) | $(KUBECTL) apply -f -
+	curl --retry $(CURL_RETRIES) -sSL https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.0/cluster-api-components.yaml | $(ENVSUBST) | $(KUBECTL) apply -f -
```

Comment:
this branch might need to be rebased, not sure why v1.2.0 changes are showing up here

Reply:
yuck